### DISCUSSION TITLE
Hide the theme switcher when theming functionality is disabled

### DISCUSSION THREAD

**Discussion by zjy365** [fuma-nama/fumadocs#965] [Q&A] [answered]
I'll create a pull request to implement this feature if it's not available yet
<img width="253" alt="image" src="https://github.com/user-attachments/assets/ac43b647-3000-4276-a39b-6bf2f0624c87">

**Comment by fuma-nama** [comment]
I think the `disableThemeSwitch` option of docs/home layouts should work

**Reply by zjy365** [reply]
Thank you, I found it. This feature was added in version 14.0.0
https://github.com/fuma-nama/fumadocs/blob/59840ed83bd34ef237d5ca44f2b7e7eeaf20ffb5/packages/ui/CHANGELOG.md?plain=1#L90

**Comment by Gitstar-OC** [comment]
I would love to see how you do it.

**Reply by fuma-nama** [reply, marked as answer]
You can always rely on TypeScript intellisense (autocomplete)
```ts
export const baseOptions: BaseLayoutProps = {
  disableThemeSwitch: true,
}
```
